### PR TITLE
Ch01-Author-Review-V2

### PR DESCRIPTION
**Pull request by rpowell22**

### CODE REVIEW

File: 01-introduction.Rmd [hunk 1]
```diff
@@ -45,3 +37,38 @@ What to expect in this book:
...
+library(osfr)
+source("helper-fun/helper-functions.R")
+
+recs_in <- read_osf("recs_2015.rds")
+anes_in <- read_osf("anes_2020.rds")
+```
+
+RECS is a household study of energy consumption in the U.S. which provides energy consumption and expenditures data. RECS funded by Energy Information Administration and collects information through energy suppliers through in-person, phone, and web interviews. It has been fielded 14 times between 1950 and 2020 and includes questions about appliances, electronics, heating, air conditioning (A/C), temperatures, water heating, lighting, energy bills, respondent demographics, and energy assistance.  Here is an overview of the data we just read in `recs_in`:
+
+```{r}
```
Comment (rpowell22) [inline 01-introduction.Rmd:60]: Right now this is a static output which works fine for PDF, but wasn't sure if we wanted to add pagination for the HTML output.  Thoughts?  (would impact ANES below as well).

Comment (szimmer) [reply 01-introduction.Rmd:60]: Perhaps using summary or glimpse rather than trying to print it?

File: 01-introduction.Rmd [hunk 2]
```diff
@@ -45,3 +37,47 @@ What to expect in this book:
 - **Chapter \@ref(c10-ambarom-vignette)**: AmericasBarometer Vignette. A vignette on how to analyze data from the AmericasBarometer, a survey of attitudes, evaluations, experiences, and behavior in countries in the Western Hemisphere. This includes how to make choropleth maps with survey estimates.
 
 Most chapters include code to follow along with.  Each chapter with this code will start with a set-up section.  This will include all of the code needed to load packages and datasets to use in the chapter.  We then provide an overview of the topic along with examples on how to use the functions.  Most chapters end with exercises to work through.  Solutions to these exercises can be found online at [XX].
```
Comment (szimmer) [inline 01-introduction.Rmd:39]: I think we will have solutions in an appendix.

File: 01-introduction.Rmd [hunk 3]
```diff
@@ -45,3 +37,47 @@ What to expect in this book:
...
+
+```{r}
+#| label: intro-recs
+recs_in
+```
+
+From this output we can see that there are `r nrow(recs_in) %>% formatC(big.mark = ",")` rows and `r ncol(recs_in) %>% formatC(big.mark = ",")` variables.  We can see that there are variables containing an ID (`DOEID`), regional information (e.g., `Region`, `MSAStatus`), along with information about the house including the type of house (`HousingUnitType`) and when the house was built (`YearMade`). Additionally, there are a long list of weighting variables that will be used in the analysis (e.g., `NWEIGHT`, `BRRWT1`).  We will discuss how to use these weighting variables in Chapter \@ref(c03-specifying-sample-designs).
```
Comment (rpowell22) [inline 01-introduction.Rmd:73]: Do we want to create codebooks for the data we create here? maybe as an appendix?  thinking for both RECS and ANES.

**Pull request merged**